GrandCentralBoard/Widgets/Bonus/Extensions/Array.swift

```diff
@@ -18,3 +18,17 @@ extension RangeReplaceableCollectionType where Generator.Element : Equatable {
     }
 
 }
+
+extension SequenceType where Generator.Element == Bonus {
+    func sortByDate() -> [Bonus] {
```

Comment:
From this name, I don't know how I'm sorting (asc/desc). `.OrderedDescending` might be an option passed to this call - WDYT?